pkg/server/server.go

```diff
@@ -958,7 +953,15 @@ func New(ctx context.Context, config *lepconfig.Config, endpoint string, cliUID
 
 	go s.updateToken(ctx, dbRW, uid, endpoint)
 
-	go func() {
+	go func(nvmlInstance nvidia_query_nvml.InstanceV2) {
+		defer func() {
```

Comment:
We can temporarily fix this. Eventually, we need to ensure that there is only one nvml instance globally, and only one place is responsible for initializing and destroying the nvml instance. This is also what we need to solve in the next version.